content/scalers/apache-kafka-topic.md

```diff
@@ -132,4 +134,59 @@ spec:
       lagThreshold: "50"
     authenticationRef:
       name: keda-trigger-auth-kafka-credential
+```
+
+Your kafka head on Azure Event Hubs - turn on sasl ssl plain auth
+
+For more details on how to get the Event Hub Namespace and Event Hub ConnectionString, refer to documentations [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quickstart-kafka-enabled-event-hubs).
```

Comment:
I don't think that a "specific" example like this should be part of the generic Kafka topic trigger documentation. Maybe we need more a section for examples.

Reply:
Event Hubs speaks Kafka so it's a fair point to note in the scaler docs here, but I'd suggest to introduce a "Using Kafka with other implementations" section and move it under there, ok for you?

Reply:
I guess in this case I will go with just providing a sample in the Samples documentation instead. Would that be ok?

Reply:
Yes, I agree with having a dedicated sample for it.